tests/ci/integration/run_crt_integration.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -exu
```

Comment:
minor but just a slight tweak I'm trying to work into our scripts to conveniently not hard-code bash but let env find it. 

```suggestion
#!/usr/bin/env bash

set -exu
```

Or this would work too:

```suggestion
#!/usr/bin/env -S bash -exu
```